src/useSubscription/models/channel.ts

```diff
@@ -75,6 +77,46 @@ export default class Channel<T extends Action = Action> {
     return Math.min(...intervals)
   }
 
+  public get paused(): boolean {
+    return this._paused
+  }
+
+  public set paused(paused: boolean) {
+    this._paused = paused
+
+    for (const subscription of this.subscriptions.values()) {
+      subscription.paused.value = paused
+    }
+
+    if (this.late) {
+      this.refresh()
+    }
```

Comment:
I'd expect that `refresh` only gets called if the subscription/channel isn't paused but this looks like it'll call the action no matter what when `paused` is set

Reply:
Otherwise I'm not totally sure what `late` is for except informational purposes, i.e. this data is known to be stale because something tried to update it but couldn't

Reply:
Hmm right, though it cannot be `late` if it wasn't paused. So I don't think this could ever be called when setting `paused: true`. But I'll add a check to make that more clear 👍

Reply:
Yeah that makes sense - thanks!

Reply:
Correct, `late` is set to true when the action is executed (either from an interval or a refresh) but the execution was prevented because the channel is paused. So late just informs us that we need to refresh when resuming.